spec/controllers/campaign_pages_controller_spec.rb

```diff
@@ -1,28 +1,77 @@
 require 'rails_helper'
-require 'capybara/poltergeist'
-require 'helper_functions'
-
-RSpec.describe CampaignPagesController, type: :feature do
-  scenario 'Redirect after a user signs the petition' do
-    cam_page = create_petition_page
-    visit "/campaign_pages/#{cam_page.id}"
-    expect(find('#redirect-location')).to have_text 'www.google.com'
-  end
 
-  scenario 'Index displays the list of active campaign pages' do
-    cam_page = create_petition_page
-    log_in
-    visit '/campaign_pages'
-    expect(page.body).to have_text cam_page.title
-  end
+RSpec.describe CampaignPagesController, type: :controller do
```

Comment:
Yeah I think there should be a commit where we go through and change them all over and make sure that the `config.disable_monkey_patching!` setting is right.